src/poliastro/plotting/__init__.py

```diff
@@ -1,4 +1,4 @@
 from .core import OrbitPlotter2D, OrbitPlotter3D
-from .static import OrbitPlotter
```

Comment:
As @vedang08 has removed the OrbitPlotter from here, @Juanlu001  do you think  in the next release, we should completely remove it?